assets/js/press-sync.js

```diff
@@ -78,7 +78,6 @@ window.PressSync = ( function( window, document, $ ) {
 	app.syncData = function( paged, objects_to_sync ) {
 
 		$.ajax({
-			async: false,
```

Comment:
This is instrumental for the "All Sync" to work correctly via the dashboard. We actually need this method to run synchronously or else the all will actually run the sync with objects out of order since they wouldn't "wait" on their previous batch to complete.